pkg/security/probe/probe.go

```diff
@@ -366,7 +366,7 @@ func (p *Probe) GetService(ev *model.Event) string {
 func (p *Probe) NewEvaluationSet(eventTypeEnabled map[eval.EventType]bool, ruleSetTagValues []string) (*rules.EvaluationSet, error) {
 	var ruleSetsToInclude []*rules.RuleSet
 	for _, ruleSetTagValue := range ruleSetTagValues {
-		ruleOpts, evalOpts := rules.NewEvalOpts(eventTypeEnabled)
+		ruleOpts, evalOpts := rules.NewBothOpts(eventTypeEnabled)
```

Comment:
I'm not sure but aren't we always using the the Linux model here, even when running the windows probe?

Reply:
basically:
`pkg/security/secl/model` is the model of the running architecture, on windows this is the windows model
`pkg/security/seclwin/model` is always the windows model, even on linux

So in pkg/security/probe it's using secl/model so windows on windows, linux on linux

Reply:
right, I forgot `pkg/security/secl/model` also contained the windows model, thanks!